core/src/storage/__init__.py

```diff
@@ -1,8 +1,7 @@
-from storage import common, device
+from storage import cache, common, device
```

Comment:
```suggestion
# make sure to import cache unconditionally on top level, so that it is imported (and retained) together with the storage module
from storage import cache, common, device
```